tests/testthat/test-base-comparison.R

```diff
@@ -0,0 +1,10 @@
+context("Assertion messages on base operators")
```

Comment:
Can you please match the context name to the file name? (As you did below)

Reply:
@hadley ugh I'm sorry, looks like I deleted my fork of `assertthat` a while ago (since I Forgot this PR was still open). I'm not sure how to re-associate this PR with a fork I can push new changes to. I tried editing in-browser to address this but Github doesn't like that either (saying "you must be on a branch").

Can we merge this and then I can make a follow-up PR to address this remaining change?

Reply:
Sure